cropper/src/main/java/com/canhub/cropper/CropImageActivity.kt

```diff
@@ -62,7 +62,7 @@ open class CropImageActivity :
 
         if (savedInstanceState == null) {
             if (cropImageUri == null || cropImageUri == Uri.EMPTY) {
-                if (CropImage.isExplicitCameraPermissionRequired(this)) {
+                if (pickImageOptions.includeCamera && CropImage.isExplicitCameraPermissionRequired(this)) {
```

Comment:
This is not something we want to remove.

If we want this always `true` (same behaviour if you remove), the user just need to make the `pick option camera == true` 

+ this do not change anything related to #271, since this is not related to permission, just a user setting